exporter/app.py

```diff
@@ -24,7 +25,7 @@
 
         logger.info("Registering collectors")
         REGISTRY.register(test_collector.TestCollector(
-            args.username, args.api_key, args.tags))
+            bool(distutils.util.strtobool(args.use_v1_uptime_endpoints)), bool(distutils.util.strtobool(args.use_v1_maintenance_windows_endpoints)), args.username, args.api_key, args.tags))
```

Comment:
You shouldn't need to use distutils then cast the result to a bool here.. Consider using optional arguments on the `TestCollector` class.